### ISSUE TITLE
About multi-gpu training

### ISSUE THREAD

**Issue by Junwen-Huang** [open-mmlab/mmpose#193]
Great job! I am trying to train with multiple GPUs with this: 
```
./tools/dist_train.sh configs/top_down/darkpose/coco-wholebody/hrnet_w48_coco_wholebody_384x288_dark_plus.py 8 --work-dir work_dir/top_down/darkpose_w48_cocowhole_gpu8 --seed 9527 --deterministic
```
I found that GPU utilization fell down to 0 after each epoch, and next epoch began after a long time (about 20 mins). In the case of single GPU, I did not found this. 
Anyone know how to fix this?

**Comment by innerlee** [comment]
if you reduce the dataset size to 1/10, will the time be reduced?

**Comment by Junwen-Huang** [comment]
> if you reduce the dataset size to 1/10, will the time be reduced?

I tried it. The time is reduced, but not reduced to 1/10 (about 13 mins).

**Comment by innerlee** [comment]
okay, at that time, could you also monitor the cpu usage by `htop`, to see if there are any background computation?

@jin-s13 any thoughts?

**Comment by Junwen-Huang** [comment]
> okay, at that time, could you also monitor the cpu usage by `htop`, to see if there are any background computation?
> 
> @jin-s13 any thoughts?

Both GPU utilization and CPU usage became high at the same time.

![CPU](https://user-images.githubusercontent.com/14028928/96551085-6027da00-12e4-11eb-8e77-a00f7a5a147e.png)
![GPU](https://user-images.githubusercontent.com/14028928/96551104-661dbb00-12e4-11eb-9a32-88235d6fa674.png)

**Comment by innerlee** [comment]
It seems that they are waiting for something. How about the disk usage? You can use `nmon` and press `d`

**Comment by Junwen-Huang** [comment]
It seems that the disk is free...
![all](https://user-images.githubusercontent.com/14028928/96574052-8b6cf200-1301-11eb-864f-4767c1190ef7.png)

**Comment by innerlee** [comment]
Hi @Junwen-Huang sorry for the late reply. Have you sloved the problem? Any further details if it persist?

**Comment by jin-s13** [comment]
This can be solved by #923
